pkg/scheduler/objects/queue_test.go

```diff
@@ -415,136 +415,6 @@ func TestAddApplication(t *testing.T) {
 	assert.Equal(t, len(leaf.applications), 1, "Application was not replaced in the queue as expected")
 }
 
-func TestAddApplicationWithTag(t *testing.T) {
```

Comment:
This is no longer relevant as we don't set resources in `AddApplication()`. The resource logic is now tested in `TestAddTGApplication()` and `TestAddTGAppDynamic()`.